src/Cron.php

```diff
@@ -8,13 +8,15 @@
 
 namespace Redbit\SimpleShop\WpPlugin;
 
+use WP_Query;
+
 class Cron {
 	/**
 	 * @var Plugin
 	 */
 	private $loader;
 
-	public function __construct(Plugin $loader) {
+	public function __construct( Plugin $loader ) {
```

Comment:
Opravdu mezery před a za??

Reply:
Je to dle: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/